API.md

```diff
@@ -10,15 +10,21 @@ const Log = require('ipfs-log')
 
 ### Constructor
 
-#### new Log(ipfs, [id])
+#### new Log(ipfs, access, identity, [id])
 
 Create a log. Each log gets a unique ID passed as an argument. Returns a `Log` instance.
```

Comment:
Should we add some description of what access and identity are?

Another possible improvement could be to change the description on "ID" to "The log's ID, the canonical address and unique identifier per log, can be passed in as an optional argument. If `id` is not given, an ID will automatically be generated." or smth like that.

Reply:
I think generally, somewhere in the docs, we should provide an example JSON for the identity so users can see which keys/values there are, how they relate and what's stored in each field.

Reply:
Related and while not really part of this PR, I think it'd be great to provide also example JSON entries in the docs (could be in API or a dedicated one, perhaps not in README) for the same reasons as for the identity as described above.

Reply:
I think that could probably go in the README; perhaps a tutorial file?